src/deploymentmanager.ts

```diff
@@ -505,6 +512,8 @@ export class DeploymentManager implements IDeploymentManager {
         data.push('PCS_EVENTHUB_NAME=' + outputs.messagesEventHubName.value);
         data.push('PCS_AUTH_REQUIRED=false');
         data.push('PCS_AZUREMAPS_KEY=static');
+        data.push('PCS_TELEMETRY_STORAGE_TYPE=' + outputs.telemetryStorageType.value);
```

Comment:
Should the AAD values be printed here as well?

Reply:
this function is only used for local deployment and there is no AAD application or secret to be created.